ApplyToBecomeInternal/ApplyToBecomeInternal/ViewModels/ProjectViewModel.cs

```diff
@@ -26,6 +26,24 @@ public ProjectViewModel(Project project)
 			ProjectDocuments = project.ProjectDocuments;
 			RationaleForProject = project.Rationale.RationaleForProject;
 			RationaleForTrust = project.Rationale.RationaleForTrust;
+			RationaleMarkAsComplete = project.Rationale.RationaleMarkAsComplete;
+			SetRationaleTaskListStatus();
+		}
+
+		private void SetRationaleTaskListStatus()
+		{
+			if (RationaleMarkAsComplete)
+			{
+				RationaleTaskListStatus = TaskListItemViewModel.Completed;
+			}
+			else if (string.IsNullOrWhiteSpace(RationaleForProject) && string.IsNullOrWhiteSpace(RationaleForProject))

```

Comment:
should this be 
`string.IsNullOrWhiteSpace(RationaleForProject) && string.IsNullOrWhiteSpace(RationaleForTrust)`?

Reply:
Haha good catch